app/test/unit/repositories-list-grouping-test.ts

```diff
@@ -3,28 +3,22 @@ import {
   KnownRepositoryGroup,
 } from '../../src/ui/repositories-list/group-repositories'
 import { Repository, ILocalRepositoryState } from '../../src/models/repository'
-import { GitHubRepository } from '../../src/models/github-repository'
-import { Owner } from '../../src/models/owner'
-import { getDotComAPIEndpoint } from '../../src/lib/api'
 import { CloningRepository } from '../../src/models/cloning-repository'
+import { gitHubRepoFixture } from '../helpers/github-repo-builder'
 
 describe('repository list grouping', () => {
   const repositories: Array<Repository | CloningRepository> = [
     new Repository('repo1', 1, null, false),
     new Repository(
       'repo2',
       2,
-      new GitHubRepository(
-        'my-repo2',
-        new Owner('me', getDotComAPIEndpoint(), null),
-        1
-      ),
+      gitHubRepoFixture({ owner: 'me', name: 'my-repo2' }),
       false
     ),
     new Repository(
       'repo3',
       3,
-      new GitHubRepository('my-repo3', new Owner('', '', null), 1),
+      gitHubRepoFixture({ owner: '', name: 'my-repo3', endpoint: '' }),
```

Comment:
I'm curious about this case of `endpoint` being an empty string... is it particularly important for this test?

Reply:
thanks for the nudge, i wondered that too, but didn't dig in...until now. 😄

it is important, only in that we need it to be a non-github dot com endpoint (its supposed to be a repo on gh enterprise instead). i've updated the test to use a more obvious "dummy" endpoint in https://github.com/desktop/desktop/pull/9206/commits/21773bb76077eb19dba9dcde233754938b1815c2.